Automattic/jetpack
# Jetpack Setup Wizard: Add questionnaire state

#### Changes proposed in this Pull Request:

* This PR builds on the work in https://github.com/Automattic/jetpack/pull/15429, https://github.com/Automattic/jetpack/pull/15501, https://github.com/Automattic/jetpack/pull/15579, and https://github.com/Automattic/jetpack/pull/15637 by adding state to Redux and updating the pages to save answers.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

#### Testing instructions:






1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit `/wp-admin/admin.php?page=jetpack#/setup`.
3. Open up Redux dev tools and navigate in the state to `jetpack.setupWizard.questionnaire`.
4. Advance through the flow choosing various answers and verify that the state changes in the dev tools.
5. Refresh the page to verify that saving/loading of the Redux state works.

#### Proposed changelog entry for your changes:

* Should be one entry for the whole Jetpack Setup Wizard project.
